src/Routes/index.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import {Switch, Route} from 'react-router-dom';
+
+import Home from '../components/Home';
+import Header from '../components/Header';
+
+const Routes =()  =>{
+    return(
+    <Switch>
+        <Route exact path ="/" component={Home}/>
```

Comment:
Can you please also add the path for a restaurant? this should include an id in the url

Reply:
done